verify/cli.go

```diff
@@ -80,23 +80,27 @@ func CLIFlags(envPrefix, category string) []cli.Flag {
 			Name:    G1PathFlagName,
 			Usage:   "Directory path to g1.point file.",
 			EnvVars: withEnvPrefix(envPrefix, "TARGET_KZG_G1_PATH"),
-			// TODO: should use absolute path wrt root directory to prevent future errors
-			//       in case we move this file around
-			Value:    "../resources/g1.point",
+			// we use a relative path so that the path works for both the binary and the docker container
+			// aka we assume the binary is run from root dir, and that the resources/ dir is copied into the working dir of the container
```

Comment:
I thought binary is run from the workdir, which is /app

Reply:
That’s the binary in the container. By binary here I just meant if we make + .//bin/… or go run